### ISSUE TITLE
Failed to load caffe layers

### ISSUE THREAD

**Issue by xuehaouwa** [speedinghzl/DSRG#17]
Hi,

I have got this error:
`F0730 17:42:12.306586  5815 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: ImageData (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Clip, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, Swish, TanH, Threshold, Tile)`

I have build caffe (make all and make pycaffe) successfully but still got this error. Any ideas?

BTW, I have uncommented `WITH_PYTHON_LAYER := 1` in the caffe make file.

MANY THANKS!!!
